app/javascript/dashboard/routes/dashboard/settings/profile/NotificationPreferences.vue

```diff
@@ -284,9 +284,9 @@ export default {
           {{ $t('PROFILE_SETTINGS.FORM.NOTIFICATIONS.BROWSER_PERMISSION') }}
         </span>
       </div>
-      <FormSwitch
-        :model-value="hasEnabledPushPermissions"
-        @update:model-value="onRequestPermissions"
+      <ToggleSwitch
+        v-model="hasEnabledPushPermissions"
+        @change="onRequestPermissions"
```

Comment:
The `change` is fine, but `@update:model-value` would work too. Just an FYI, feel free to resolve this